### PR TITLE
PXC-3138: Upgrade - PXC 5.7 nodes

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/howtos/upgrade_guide.rst [hunk 1]
```diff
@@ -42,7 +42,27 @@ If downtime is acceptable, shut down the cluster and upgrade all nodes to |pxc|
...
+.. seealso::
+
+   |MySQL| Documentation: super_read_only system variable
+      https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_super_read_only
+
+.. important:: 
+
+   If writes are directed to a |pxc| 8.0 node in a cluster that has |pxc| 5.7,
+   the cluster will be broken down. In this case, the system reports the
+   following error message:
```
Comment (altmannmarcelo) [inline doc/source/howtos/upgrade_guide.rst]: The message can vary depending on the query. So I would adjust the working to something like:

In this case, the system reports a message like below:

File: doc/source/howtos/upgrade_guide.rst [hunk 2]
```diff
@@ -42,7 +42,27 @@ If downtime is acceptable, shut down the cluster and upgrade all nodes to |pxc|
...
+
+   .. code-block:: text
+
+      [ERROR] Slave SQL: Error 'Character set '#255' is not a compiled character set and
+      is not specified in the '/usr/share/percona-xtradb-cluster/charsets/Index.xml' file' on
+      query. Default database: 'test'. Query: 'BEGIN', Error_code: 22
```
Comment (altmannmarcelo) [inline doc/source/howtos/upgrade_guide.rst]:
```suggestion
      query. Default database: '[DB NAME]'. Query: '[QUERY]', Error_code: 22
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: doc/source/howtos/upgrade_guide.rst [hunk 3]
```diff
@@ -42,7 +42,27 @@ If downtime is acceptable, shut down the cluster and upgrade all nodes to |pxc|
 
 If downtime is not possible, the rolling upgrade is supported but ensure the
 traffic is controlled during the upgrade and writes are directed only to 5.7
-nodes until all nodes are upgraded to 8.0.
+nodes until all nodes are upgraded to |version|.
+
+With |pxc| 5.7 nodes in the cluster, start |pxc| |version| nodes with
+``super_read_only`` enabled.
```
Comment (altmannmarcelo) [inline doc/source/howtos/upgrade_guide.rst]: remove until PXC-3189 is completed

**Review by kathy-williamson — APPROVED** [review]
I would request that we add another ticket to research how to make this message a meaningful message for the situation

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**